TLM/TLM/UI/SubTools/LaneConnectorTool.cs

```diff
@@ -1040,14 +1062,15 @@ public LaneConnectorTool(TrafficManagerTool mainTool)
                         && ((laneInfo.m_vehicleType & LaneConnectionManager.VEHICLE_TYPES)
                             != VehicleInfo.VehicleType.None)) {
                         if (connManager.GetLaneEndPoint(
-                            segmentId,
-                            startNode,
-                            laneIndex,
-                            laneId,
-                            laneInfo,
-                            out bool isSource,
-                            out bool isTarget,
-                            out Vector3? pos)) {
+                            segmentId: segmentId,
+                            startNode: startNode,
+                            laneIndex: laneIndex,
+                            laneId: laneId,
+                            laneInfo: laneInfo,
+                            outgoing: out bool isSource,
+                            incoming: out bool isTarget,
+                            pos: out Vector3? pos))
```

Comment:
a big chunk of changes in this PR is auto code cleanup changes in places where you have not made any other modifications. in future PR is it possible to do it in two commits (one commit for auto code cleanup and another for the functional changes to the code). I am not sure if its practical because you do not know in advance which files you are going to modify.

I am thinking who is the one to speak! in my own PR's my commits are a big mess!

Reply:
I'm touching things i visit, not everything, and the changes are not too many i hope.
Just scroll through them :)